pkgs/development/tools/build-managers/wafHook/default.nix

```diff
@@ -1,9 +1,9 @@
-{ lib, stdenv, pkgs, python, makeSetupHook, waf }:
+{ lib, stdenv, pkgs, python3, makeSetupHook, waf }:
```

Comment:
Should this go to the `callPackage`?

Reply:
I too tend to agree with this approach - perhaps we should have 2 different `wafHook`s - one that's called with default `python` and the other with `{ python = python3 }`.